snapshot-labs/snapshot
# feat: Add subscription form to email report

### Issues

Fixes https://github.com/snapshot-labs/envelop/issues/35

This PR add a form to subscribe to snapshot email notifications, from the user account menu on the header

### Changes 

1. Add email form to subscribe to envelop email notifications 

### How to test

1. Log in to your account
2. Click on your profile in the header navigation
3. In the dropdown, select "SUBSCRIBE"
4. Fill the email form in the modal
5. Submit
6. Sign the message
7. See success/error message

### To-Do

- [x] Show the form in relevant places 
- [x] Add modal/message when subscription is successful
- [x] Add error message on error


### Self-review checklist
